packages/nearfi/src/lib/nearfi.ts

```diff
@@ -166,8 +166,8 @@ const NearFi: WalletBehaviourFactory<InjectedWallet> = async ({
     },
 
     async verifyOwner({ message }) {
-      logger.log("NearFi:verifyOwner", { message });
-
+      logger.log("NearSnap:verifyOwner", { message });
```

Comment:
This PR is about `Near Snap` why is the change made in the `NearFi` wallet?

Please revert this change.